terraform/aptos-node/gcp/kubernetes.tf

```diff
@@ -113,6 +113,24 @@ resource "helm_release" "logger" {
   }
 }
 
+resource "helm_release" "vector_log_agent" {
+  count            = var.enable_vector_log_agent ? 1 : 0
+  name             = "${local.helm_release_name}-vector-logs"
+  chart            = local.vector_log_agent_helm_chart_path
+  max_history      = 5
+  namespace        = "vector"
+  create_namespace = true
+  wait             = false
+
+  values = var.vector_log_agent_helm_values
```

Comment:
`var. vector_log_agent_helm_values` as a list of strings is actually pretty good, since helm will handle the values merging for us in order. We might want to consider using this pattern for all our Aptos helm releases too, instead of just passing a map of values